server/constants/constants.go

```diff
@@ -47,6 +47,8 @@ const (
 	// URL query params constants
 	QueryParamProject   = "project"
 	QueryParamChannelID = "channel_id"
+	QueryParamOffset    = "offset"
+	QueryParamLimit     = "limit"
```

Comment:
I think we should take "page" and "per_page" as the query params as they are more simple from a user's point of view or even a frontend dev's POV.

Reply:
Sure, sounds good for FE dev
but on the server, we can use offset and limit for betterment